javers-persistence-sql/src/main/java/org/javers/repository/sql/schema/FixedSchemaFactory.java

```diff
@@ -39,6 +39,7 @@
     public static final String SNAPSHOT_COMMIT_FK =    "commit_fk";
     public static final String SNAPSHOT_GLOBAL_ID_FK = "global_id_fk";
     public static final String SNAPSHOT_TYPE =         "type";
+    public static final String VERSION =               "version";
```

Comment:
I dont have more commments,
besides the old one to CdoSnapshotTypeAdapterTest.groovy which is not applied yet?

Considering your second question *which is the field that is the same for all snapshots of a given entity.*
There are no special columns in SQL tables which are not reflected in Java (besides primary and foreign keys).

If you are thinking about SQL for data migration script,
snapshots can be sorted asc by primary key and updated: `SET version = rownum` 
But rownum is dialect-specific. In Oracle - `rownum`, in Postgres -  `row_number()` and so on ...

I think that troubles with data migration script should not stop us from releasing `getVersion()`
as soon as we prepare clear javadoc 
(that will describe the issue with nulls in old snapshots versions to JaVers users)